docker/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG base
-FROM golang:1.12.4-alpine as builder
+FROM golang:1.13.8-alpine as builder
```

Comment:
Maybe we can use golang 1.14, which is the newest golang version?

Reply:
We haven't throughly tested the node with 1.14 yet. We can probably do more testing before upgrade the version.